changelog.md

```diff
@@ -2,7 +2,52 @@
 
 ## 2.8.0~rc1
 
+### Web applications
+* Updated strings based on translator feedback (#7057)
+* Improved redwood stream performance and testing (#7070)
+* Dependency changes:
+  * openssl rust crate from 0.10.57 to 0.10.60 (#7083)
+  * cryptography from 41.0.3 to 41.0.7 (#7086)
+  * rustix rust crate from 0.38.18 to 0.38.21 (#7114)
```

Comment:
I think the more important one is dba0af69191b7ff81ef2812850e83e0de9f6641d, which upgraded is-terminal so that it no longer depends on rustix (which is now a dev-dep only).

Reply:
:+1: